### DISCUSSION TITLE
mongodb install problem

### DISCUSSION THREAD

**Discussion by Luis981123** [DenverCoder1/github-readme-streak-stats#651] [Q&A] [answered]
How to install mongodb on ubuntu 22.0.4?

**Comment by Arturs1123** [comment, marked as answer]
https://www.cherryservers.com/blog/install-mongodb-ubuntu-22-04#step-1-install-mongodb-on-ubuntu-2204
Please check it

**Reply by Luis981123** [reply]
Thank you!!!
